examples/viewer.html

```diff
@@ -239,7 +237,7 @@
       this.container.appendChild(this.iframe);
 
       // Notice that name can only be set once the IFrame is in the DOM.
-      this.iframe.contentWindow.name = SENTINEL + paramsStr(params);
+      this.iframe.contentWindow.name = APP + paramsStr(params);
```

Comment:
These are actually two unrelated constants. I don't see a strong reason for them to be different, but currently they are also different.

That being said, however, please remove this line completely - it's unnecessary.